cms/taxonomy/models.py

```diff
@@ -53,18 +53,21 @@ class Meta:
         index.AutocompleteField("title"),
     ]
 
-    def save_new_topic(self, parent_topic: Optional["Topic"] = None) -> None:
+    @classmethod
+    def create_under_parent(cls, topic: "Topic", parent_topic: Optional["Topic"] = None) -> None:
```

Comment:
I think that using the word "create" is somewhat confusing here. We don't _create_ the topic, because both in the management command and in the tests the topic _already exists_ when this method is called.

Something like `append_to_tree`, `insert_child` or `append_node` (probably best) would make more sense.

Reply:
I was following the ticket under the impression there was some consensus on this, but if that isn't the case I'm happy to change it. I would have done this in an overriding `save` method, only that doesn't work because `add_child` calls `save` itself.

Reply:
My preferred naming would still be `Topic.save_new(...)`, I'll revert to that for now and see what reviewers think.